src/inventory/dispense/dispense-page.jsx

```diff
@@ -71,53 +97,84 @@ export const DispensePage = () => {
 				stickyHeader={true}
 			>
 				{({ rows, headers, getTableProps, getHeaderProps, getRowProps }) => (
-					<TableContainer>
-						<TableToolbar style={{ width: "15rem" }}>
-							<TableToolbarContent style={{ justifyContent: "flex-start" }}>
-								<TableToolbarSearch
-									value={searchText}
-									onChange={handleSearch}
-								/>
-							</TableToolbarContent>
-						</TableToolbar>
-						<Table {...getTableProps()}>
-							<TableHead>
-								<TableRow>
-									{headers.map((header) => (
-										<TableHeader
-											{...getHeaderProps({
-												header,
-												isSortable: true,
-											})}
-										>
-											{header.header}
-										</TableHeader>
-									))}
-								</TableRow>
-							</TableHead>
-							<TableBody>
-								{rows.map((row) => (
-									<TableRow
-										{...getRowProps({ row })}
-										onClick={() => handleRowClick(row)}
-									>
-										{row.cells.map((cell) => (
-											<TableCell key={cell.id}>
-												{" "}
-												{cell.info.header === "patientName" ? (
-													<a href="#">{cell.value}</a>
-												) : (
-													cell.value
-												)}
-											</TableCell>
+					<>
+						<TableContainer>
+							<TableToolbar style={{ width: "15rem" }}>
+								<TableToolbarContent style={{ justifyContent: "flex-start" }}>
+									<TableToolbarSearch
+										value={searchText}
+										onChange={handleSearch}
+									/>
+								</TableToolbarContent>
+							</TableToolbar>
+							<Table {...getTableProps()}>
+								<TableHead>
+									<TableRow>
+										{headers.map((header) => (
+											<TableHeader
+												{...getHeaderProps({
+													header,
+													isSortable: isSortable(header.key),
+												})}
+											>
+												{header.header}
+											</TableHeader>
 										))}
 									</TableRow>
-								))}
-							</TableBody>
-						</Table>
-					</TableContainer>
+								</TableHead>
+								<TableBody>
+									{rows.map((row) => (
+										<TableRow
+											{...getRowProps({ row })}
+											// onClick={() => handleRowClick(row)}
+										>
+											{row.cells.map((cell) => (
+												<TableCell key={cell.id}>
+													{" "}
+													{cell.info.header === "patientName" ? (
+														<Link
+															href="#"
+															onClick={() => handleOnLinkClick(row)}
+														>
+															{cell.value}
+														</Link>
+													) : (
+														cell.value
+													)}
+												</TableCell>
+											))}
+										</TableRow>
+									))}
+								</TableBody>
+							</Table>
+						</TableContainer>
+					</>
 				)}
 			</DataTable>
+			{showModal && (
+				<CustomModal
+					data={{
+						id: 1,
+						name: "Patty O'Furniture",
```

Comment:
This hardcoding is because a later story will show this information from backend?

Reply:
yes , will be taken care when playing dispense screen UI card